test/unit/visual/cases/webgl.js

```diff
@@ -34,4 +34,20 @@ visualSuite('WebGL', function() {
       screenshot();
     });
   });
-});
+
+  visualSuite('3DModel', function() {
+    visualTest('OBJ model with MTL file displays diffuse colors correctly', function(p5, screenshot) {
+      return new Promise(resolve => {
+        p5.createCanvas(50, 50, p5.WEBGL);
+        p5.loadModel('unit/assets/octa-color.obj', model => {
```

Comment:
Right now the screenshot looks like just a dot. Maybe to make this clearer we can call `model.normalize()` to normalize its size to, I think, 200px?

Reply:
Maybe we can also add one more test importing an object with no colors, and then drawing it with a colored `fill()` to show that it can take on new colors.